src/libs/API/parameters/ExportSearchWithTemplateParams.ts

```diff
@@ -0,0 +1,12 @@
+import type {SearchQueryString} from '@components/Search/types';
+
+type ExportSearchWithTemplateParams = {
+    templateName: string;
+    templateType: string;
+    jsonQuery: SearchQueryString;
+    reportIDList: string[];
+    transactionIDList: string[];
```

Comment:
Should this be `string`? Are we imploding it before making the API request?

Reply:
Ah weird, I was going off of the params in `ExportSearchItemsToCSVParams` ([App params](https://github.com/Expensify/App/blob/main/src/libs/API/parameters/ExportSearchItemsToCSVParams.ts), [ExportSearchToCSV params](https://github.com/Expensify/Web-Expensify/blob/0d5b88bc0b839b974653a15c93de6c036bb67f32/api.php#L5154-L5155) but yeah maybe it should be a string 🤔

Reply:
The data is passed as a string in the request so this seems fine.